src/IR/IR.hs

```diff
@@ -488,13 +488,12 @@ instance Pretty (Expr ()) where
   pretty (Prim (CCall s) es _) =
     pretty "$" <> pretty s <> parens (hsep $ punctuate comma $ map pretty es)
   pretty (Prim (FfiCall s) es _) = pretty s <+> hsep (map (parens . pretty) es)
-  pretty (Prim (CQuote s) [] _) = pretty "$$" <> pretty s <> pretty "$$"
+  pretty (Prim (CQuote  s) [] _) = pretty "$$" <> pretty s <> pretty "$$"
 
   -- pretty (Prim Return [e] _        ) = pretty "return" <+> braces (pretty e)
   pretty (Prim p _ _) = error "Primitive expression not well-formed: " $ show p
-  pretty (Exception et _) = pretty "exception" <+> prettyet et
-    where 
-      prettyet (ExceptDefault l) = pretty l
+  pretty (Exception et _       ) = pretty "exception" <+> prettyet et
```

Comment:
Change to `"__exception"` to make it clear this is a primitve

Reply:
Done.